libs/langchain-mongodb/tests/integration_tests/conftest.py

```diff
@@ -19,13 +21,8 @@ def technical_report_pages() -> List[Document]:
 
 
 @pytest.fixture
-def connection_string() -> str:
-    return os.environ["MONGODB_URI"]
-
-
-@pytest.fixture
-def client(connection_string: str) -> MongoClient:
-    return MongoClient(connection_string)
+def client() -> MongoClient:
+    return MongoClient(CONNECTION_STRING)
```

Comment:
What's the new best-practice here? close the client?  Should we yield then close?

Reply:
We can't use fixtures because the tests are relying on properties.  So we're stuck with the warnings at shutdown.